code/modules/mob/living/silicon/mommi/mommi_subtypes/sammi.dm

```diff
@@ -0,0 +1,211 @@
+/mob/living/silicon/robot/mommi/sammi
+	name = "Stationary Assembler MMI"
+	real_name = "Stationary Assembler MMI"
+	icon = 'icons/mob/robots.dmi'
+	icon_state = "sammi_offline"
+	maxHealth = 60
+	health = 60
+	keeper=0 // 0 = No, 1 = Yes (Disables speech and common radio.)
+	prefix = "Stationary Assembler MMI"
+	canmove = 0
+	//..()
+
+
+/mob/living/silicon/robot/mommi/sammi/emag_act(mob/user)
+	if(user == src && !emagged)
+		if(module)
+			var/obj/item/weapon/robot_module/mommi/mymodule = module
+			to_chat(user, "<span class='warning'>[mymodule.ae_type] safety override initiated.</span>")
+		playsound(src, 'sound/machines/buzz-sigh.ogg', 50, 0)
+
+		var/hold = new sammiemag_base_law_type
+		src.laws.inherent = hold
+		src.show_laws()
+
+		return TRUE
+
+/mob/living/silicon/robot/mommi/sammi/update_canmove()
+	return 0
+
+/mob/living/silicon/robot/mommi/sammi/attackby(obj/item/W, mob/user)
+
+
+	if(istype(W, /obj/item/stack/cable_coil) && wiresexposed)
```

Comment:
Check if there is any fireloss to fix

Reply:
@MadmanMartian 
I figured a lot has changed just with me messing around on the production server for a few minutes.

This whole thing wasn't really intended to be merged right out of the gate, I know better than that these days, lol. 

I appreciate the notes. We'll get this shined up and see if it makes sense / can add something fun to the server.

Reply:
Glad to hear it